cmd/server-main.go

```diff
@@ -75,16 +75,20 @@ EXAMPLES:
   1. Start minio server on "/home/shared" directory.
      {{.Prompt}} {{.HelpName}} /home/shared
 
-  2. Start distributed minio server on an 32 node setup with 32 drives each, run following command on all the nodes
+  2. Start minio server on a single server with 16 drives
```

Comment:
is this added back? we on purpose removed lots of examples - we don't need to mention or encourage single node erasure-coded setups. did @abperiasamy ask you add it back?

Reply:
yeah @abperiasamy wanted me to add this.